changelog/v2.4.0-rc1/features/2707--arrow-import.md

```diff
@@ -0,0 +1,2 @@
+VAST now imports Arrow IPC data, which is the same format it already
+supports for export.
```

Comment:
Please also add documentation before merging here: https://vast.io/docs/use/ingest

Reply:
Added some docs, also for `export arrow`. A quick pass over that is appreciated. (maybe @mavam instead)?

Reply:
resolving as @mavam added a commit dubbed "fine-tune arrow docs", so I assume it's fine.